lib/modules/minifyCss.es6

```diff
@@ -51,3 +61,13 @@ function processStyleAttr(node, cssnanoOptions) {
             );
         });
 }
+
+function stripCdata(css) {
+    const leftStrippedCss = css.replace('<![CDATA[', '');
+    if (leftStrippedCss === css) {
+        return css;
+    }
+
+    const strippedCss = leftStrippedCss.replace(']]>', '');
+    return leftStrippedCss === strippedCss ? css : strippedCss;
+}
```

Comment:
It is possible to have `CDATA` inside `<style>` node, see [Styling - SVG 1.1 - W3C](https://www.w3.org/TR/SVG11/styling.html).